npm/private/test/links_defs_checked.bzl

```diff
@@ -0,0 +1,258 @@
+                       "@aspect_rules_js//platforms/cpu:x86_64": ["--arch=x64"],
+                       "//conditions:default": [],
+                   }),
+            copy_srcs_to_bin = False,
+            tool = "@aspect_rules_js//npm/private/lifecycle:lifecycle-hooks",
+            out_dirs = ["node_modules/.aspect_rules_js/rollup@2.70.2/node_modules/rollup"],
+            tags = ["manual"],
+            execution_requirements = {
+                "no-sandbox": "1",
+            },
+            mnemonic = "NpmLifecycleHook",
+            progress_message = "Running lifecycle hooks on npm package rollup@2.70.2",
+            env = {},
+        )
+
+        # post-lifecycle npm_package
+        _npm_package_internal(
+            name = "{}/pkg_lc".format(store_target_name),
+            src = ":{}/lc".format(store_target_name),
+            package = "rollup",
+            version = "2.70.2",
+            tags = ["manual"],
+        )
+
+# Generated npm_package_store and npm_link_package_store targets for npm package rollup@2.70.2
+# buildifier: disable=function-docstring
+def npm_link_imported_package_store(name):
+    link_packages = {
+        "examples/npm_deps": ["rollup"],
+    }
+    if native.package_name() in link_packages:
+        link_aliases = link_packages[native.package_name()]
+    else:
+        link_aliases = ["rollup"]
+
+    link_alias = None
+    for _link_alias in link_aliases:
+        if name.endswith("/{}".format(_link_alias)):
+            # longest match wins
+            if not link_alias or len(_link_alias) > len(link_alias):
+                link_alias = _link_alias
+    if not link_alias:
+        msg = "name must end with one of '/{{ {link_aliases_comma_separated} }}' when called from package 'rollup'; recommended value(s) are 'node_modules/{{ {link_aliases_comma_separated} }}'".format(link_aliases_comma_separated = ", ".join(link_aliases))
+        fail(msg)
+
+    link_root_name = name[:-len("/{}".format(link_alias))]
+    store_target_name = ".aspect_rules_js/{}/rollup@2.70.2".format(link_root_name)
+
+    # terminal package store target to link
+    _npm_link_package_store(
+        name = name,
+        package = link_alias,
+        src = "//:{}".format(store_target_name),
+        visibility = ["//visibility:public"],
+        tags = ["manual"],
+        use_declare_symlink = select({
+            "@aspect_rules_js//js:allow_unresolved_symlinks": True,
+            "//conditions:default": False,
+        }),
+    )
+
+    # filegroup target that provides a single file which is
+    # package directory for use in $(execpath) and $(rootpath)
+    native.filegroup(
+        name = "{}/dir".format(name),
+        srcs = [":{}".format(name)],
+        output_group = _utils.package_directory_output_group,
+        visibility = ["//visibility:public"],
+        tags = ["manual"],
+    )
+
+    return [":{}".format(name)] if True else []
```

Comment:
Where does the `if True else []` come from?

Reply:
Looks like you found it below